test/frontend/test_node.py

```diff
@@ -166,14 +166,14 @@ def do_test_expected(self):
 ("test_concat", tf.concat, "concat", [[get_rnd([1, 10]),get_rnd([10, 10]),get_rnd([20, 10])], 0], {}),
 ("test_bias_add_nchw", tf.nn.bias_add, "BiasAdd", [get_rnd([10, 32, 10, 10]),get_rnd([32])], {"data_format":"NCHW"}),
 ("test_bias_add_nhwc", tf.nn.bias_add, "BiasAdd", [get_rnd([10, 10, 10, 32]),get_rnd([32])], {"data_format":"NHWC"}),
+("test_strided_slice", tf.strided_slice, "StridedSlice", [get_rnd([5, 5]), [0, 0], [1, 5], [1, 1]], {}),
```

Comment:
We should still place these ops under version based guards.
Since we want to maintain backward compatibility as much as possible.

Reply:
I will add in comment for the reason to run those unit test only when opset version >= 9, while the opset version for the operator is actually 1.